### ISSUE TITLE
Feature - focus on next element after TextInput

### ISSUE THREAD

**Issue by jenweber** [thefrontside/bigtest#818]
This issue requires team discussion before it should be worked on.

In order to make it easier to test using keyboard navigation through a form, there should be an easy way to bring focus to the next element, just like in real life.

Today, the `fillIn` helper calls `blur()` when it is finished:

https://github.com/thefrontside/bigtest/blob/4d7c43f9daeb8be2b9343ed8ef6acc1252bc490d/packages/interactor/src/fill-in.ts#L63

This means that no elements have focus when it is complete, unlike real life when it would still be in focus until someone clicked or tabbed away.

It is helpful to call blur after fillIn, because very often things like input validation only run when focus leaves, and devs may expect that behavior from their experience with other test libraries. However, this does not work great when testing keyboard navigation.

## Some options

- Add a new action to TextInput such as `enterText`, which moves focus to the next element when it is finished
- pass an option to `fillIn` - not ideal because options are kind of cumbersome to repeat for every text input
- Any others?

To-do:
- Decide on the API
- Write a helper for finding the next element that can receive focus
- Write an action or option and use the helper to apply focus to the next element instead of blurring

**Comment by jenweber** [comment]
@cowboyd please review and add more ideas/guidance if you have them!

**Comment by cowboyd** [comment]
Looks like we can perhaps use this as something to help us out, although I'd prefer a ponyfill to a polyfill

 https://github.com/awesomeeng/FocusTraversalAPI

**Comment by jnicklas** [comment]
An idea might be to have something like a global `Keyboard` interactor, which could be used for more advanced keyboard actions. For example:

``` typescript
// there is some API to set keyboard layout
Keyboard.setLayout(DvorakLayout);

// can press individual keys, input is sent to currently focused element
await Keyboard.press('f', { meta: true }); // sends key: "f" | code: "KeyY" since the "f" on dvorak is in the "y" location of qwerty

// should move focus to the next element
await Keyboard.press('tab') // sends key='Tab' | code='Tab';

// some way to type strings of text
await Keyboard.type('Hello');

// maybe with a simple DSL for entering control characters?
await Keyboard.type('Hello<shift+enter><esc>');
```
